chrome/popup.html

```diff
@@ -1,16 +1,21 @@
 <!doctype html>
 <html>
+
 <head>
   <meta charset="utf-8">
   <link rel="stylesheet" href="popup.css">
   <script src="popup.js"></script>
 </head>
+
 <body>
   <p>Duplicate tab shortcut: <a id="duplicate-tab-shortcut" href="chrome://extensions/configureCommands">error</a>.</p>
   <p>Duplicate to new window shortcut: <a id="duplicate-to-new-window-shortcut" href="chrome://extensions/configureCommands">error</a>.</p>
+  <p>Move to new window shortcut: <a id="move-to-new-window-shortcut" href="chrome://extensions/configureCommands">error</a>.</p>
```

Comment:
In this file, can you undo the other style changes and keep only this line?

Reply:
Fixed in c9d46c0.